ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/details/tags/_cq_dialog/.content.xml

```diff
@@ -50,6 +50,13 @@
                                             name="./hideLabel"
                                             text="Hide Label"
                                             value="true"/>
+                                    <displayComputedProperty
```

Comment:
Samea s in Model, not sure why this is needed.

If we DID need it, the node name should be lowercase w/ dashes though per jackrabbit standards (property names are camelCase, not node names)

Reply:
we are going to remove it :)